src/app/app.js

```diff
@@ -681,6 +682,12 @@ nw.App.on('open', function (cmd) {
     }
 });
 
+win.on('minimize', function () {
+    if (Settings.minimizeToTray) {
+        minimizeToTray();
+    }
```

Comment:
shouldn't we stop events or something here ?
just reading through that patch makes me think we're loosing functionality.... are we sure it works on windows, osx... ?